src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_CollectionExpression.cs

```diff
@@ -782,7 +1019,8 @@ private BoundExpression RewriteCollectionExpressionElementExpression(BoundNode e
             ArrayBuilder<BoundLocal> rewrittenExpressions,
             int numberIncludingLastSpread,
             ArrayBuilder<BoundExpression> sideEffects,
-            Action<ArrayBuilder<BoundExpression>, BoundExpression, BoundExpression> addElement)
+            Action<ArrayBuilder<BoundExpression>, BoundExpression, BoundExpression> addElement,
+            Func<ArrayBuilder<BoundExpression>, BoundExpression, BoundCollectionExpressionSpreadElement, BoundExpression, bool>? tryOptimizeSpreadElement = null)
```

Comment:
>tryOptimizeSpreadElement = null [](http://example.com/codeflow?start=129&length=31)

All callers pass a non-null value. The parameter can be changed to non-optional and non-nullable.